src/main/java/com/salesforce/mirus/config/SourceConfig.java

```diff
@@ -43,8 +44,23 @@ public String getTopicsRegex() {
     return simpleConfig.originalsWithPrefix("consumer.");
   }
 
-  public String getDestinationBootstrapServers() {
-    return simpleConfig.getString(SourceConfigDefinition.DESTINATION_BOOTSTRAP_SERVERS.key);
+  private Map<String, Object> getDestinationProperties() {
+    // handle destination.bootstrap.server separately
```

Comment:
Kept this here so that these changes are backward compatible. See Line 53 which overrides only if its equivalent `destination.consumer.` prop is not defined.

Reply:
I think we should add a comment making it clear we're doing this to maintain backward compatibility.